code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -118,6 +118,28 @@
 		"replicant_alloy" = max(MAX_COMPONENTS_BEFORE_RAND - LOWER_PROB_PER_COMPONENT*clockwork_component_cache["replicant_alloy"], 1), \
 		"hierophant_ansible" = max(MAX_COMPONENTS_BEFORE_RAND - LOWER_PROB_PER_COMPONENT*clockwork_component_cache["hierophant_ansible"], 1)))
 
+/proc/clockwork_say(atom/movable/AM, message, whisper=FALSE)
+	// When servants invoke ratvar's power, they speak in ways that non
+	// servants do not comprehend.
+	// Our ratvarian chants are stored in their ratvar forms in code
+	// this changes them to english so servants can understand.
+	// Non servants will only hear the original rot13.
+	message = rot13(message)
+
+	var/list/spans = list(SPAN_ROBOT)
+
+	var/old_languages_spoken = AM.languages_spoken
+	AM.languages_spoken = list(RATVAR)
```

Comment:
Will this work? Aren't languages supposed to be bitflags instead of list elements?